distribution/osbuild-composer-clouddot-template.yml

```diff
@@ -0,0 +1,79 @@
+apiVersion: v1
+kind: Template
+labels:
+  app: osbuild-composer
+  template: osbuild-composer
+metadata:
+  annotations:
+    description: OCP template for osbuild-composer in cloud.redhat.com
+  name: osbuild-composer
+objects:
+
+- apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    labels:
+      service: osbuild-composer
+    name: osbuild-composer
+  spec:
+    replicas: 2
+    selector:
+      matchLabels:
+        name: osbuild-composer
+    strategy:
+      # Update pods 1 at a time
+      type: RollingUpdate
+      rollingUpdate:
+        # Create at most 1 extra pod over .spec.replicas
+        maxSurge: 1
+        # At all times there should be .spec.replicas available
+        maxUnavailable: 0
+    template:
+      metadata:
+        labels:
+          name: osbuild-composer
+      spec:
+        containers:
+        - image: "${IMAGE_NAME}:${IMAGE_TAG}"
+          name: osbuild-composer
+          ports:
+          - name: api
+            containerPort: 443
+            protocol: TCP
+          - name: workers
+            containerPort: 8700
+          volumeMounts:
+          - name: composer-config
+            mountPath: "/etc/osbuild-composer"
+            readOnly: true
+        volumes:
+        - name: composer-config
+          secret:
+            secretName: composer-config
+
+- apiVersion: v1
+  kind: Service
+  metadata:
+    labels:
+      service: osbuild-composer
+    name: osbuild-composer
+  spec:
+    ports:
+      - name: composer-api
```

Comment:
I think we also want a port for workers because they will also live outside the OpenShift cluster.

Reply:
Indeed, I'm not 100% sure what it'll look like, but I think that'll probably end up being different openshift service.

The problem with composer exactly those 2 ports, as I don't think we'll get cloud.redhat.com:8700/..., so we might need to be clever with routing where `..:443/api/osbuild-composer/workers/....` gets redirected to  `:8700` in the container. We'll probably need a different service for that but not 100% sure.

Reply:
okay, let's worry about that later. :) We might even want to merge these APIs to one port in composer.